.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -42,3 +45,9 @@ jobs:
         with:
           name: protractor-screenshots
           path: /home/runner/work/oppia/protractor-screenshots
+      - name: Uploading protractor-video as Artifacts
```

Comment:
Should we add this as an action and reuse it in all the places?

Reply:
Maybe action: `upload-e2e-artifacts`

[ref](https://github.com/oppia/oppia/pull/12440/files#diff-4c981b81a7c9eb78b105571de97ef300d0616557a1676e7d5c432cc66bdbbd16R1)

Reply:
I think I should do this in a different PR since this PR has a lot of changes already.

Reply:
This will help you avoid having too many changes, right? (like making a reusable action in all the places?) [not sure!]

Reply:
Since this PR is high priority, I think we should just get this merged.

Reply:
Sure, I'm fine with that! Can you please create an issue for the same? (We can assign it to new contributors.)